core/include/optee_msg.h

```diff
@@ -55,6 +55,11 @@
 #define OPTEE_MSG_ATTR_TYPE_TMEM_INPUT		0x9
 #define OPTEE_MSG_ATTR_TYPE_TMEM_OUTPUT		0xa
 #define OPTEE_MSG_ATTR_TYPE_TMEM_INOUT		0xb
+/*
+ * Special parameter type denoting that command buffer continues on
+ *  specified page
+ */
+#define OPTEE_MSG_ATTR_TYPE_NEXT_FRAGMENT	0xc
```

Comment:
This together with `OPTEE_MSG_ATTR_NESTED` is quite complicated. To make it easier to understand please describe how they can be used in one or two example use cases.